### DISCUSSION TITLE
Error running RV after successful build on Rocky Linux 8

### DISCUSSION THREAD

**Discussion by beatreichenbach** [AcademySoftwareFoundation/OpenRV#382] [Build Issues]
I'm getting this error after a successful build on Rocky Linux 8. (If everything goes well, I'll provide a docker setup as an easy reproducible environment.)

```
Version 2.0.0 (RELEASE), built on Jan 31 2024 at 22:21:42 (HEAD=2311adc).
Copyright Contributors to the Open RV Project
[419704:419704:0131/190745.113471:ERROR:extension_system_qt.cpp(121)] Failed to parse extension manifest.
Segmentation fault (core dumped)
```

I wasn't able to build RV with built-in qt5-devel packages but with the online installer (https://github.com/AcademySoftwareFoundation/OpenRV/discussions/102#discussioncomment-5385768) everything worked fine.

Any ideas on what could cause this? Do we need Qt installed on the machine that is trying to launch the application? In theory all required libraries should be included in `/lib` correct?

Thanks!

**Comment by rogernelson** [comment]
Hello, thanks for the report! One thing that can help determine the cause, if you have the time, would be to build it debug and then launch rv with the `-d` option.  This will run it in the debugger.  Once it crashes you can type `where` or `bt` to get a stack trace showing where it crashed, which should help narrow down the problem.

**Comment by beatreichenbach** [comment]
Thank you, I was able to get this resolved. See https://github.com/AcademySoftwareFoundation/OpenRV/pull/384
